api_tests/src/images.spec.ts

```diff
@@ -0,0 +1,54 @@
+jest.setTimeout(120000);
+
+import { UploadImage, DeleteImage } from "lemmy-js-client";
+import {
+  alpha,
+  setupLogins,
+  unfollowRemotes,
+} from "./shared";
+import fs = require('fs');
+
+beforeAll(async () => {
+  await setupLogins();
+});
+
+afterAll(async () => {
+  await unfollowRemotes(alpha);
+});
+
+test("Upload image and delete it", async () => {
+    // upload test image
+    // TODO: this doesnt require separate auth anymore (same for delete)
+    let upload_image = fs.readFileSync("test.png");
+    let upload_form: UploadImage = {
+        image: upload_image,
+        auth: undefined,
+    };
+    let upload = await alpha.uploadImage(upload_form);
+    console.log(upload);
+    expect(upload.files![0].file).toBeDefined();
+    expect(upload.files![0].delete_token).toBeDefined();
+    expect(upload.url).toBeDefined();
+    expect(upload.delete_url).toBeDefined();
+
+    // ensure that image download is working. theres probably a better way to do this
+    var downloadFileSync = require('download-file-sync');
```

Comment:
Any reason you're importing this with `require` inside this test instead of importing it at the top like the rest of the dependencies?

Reply:
If I move it to the other imports at the top it throws `error TS7016: Could not find a declaration file for module 'download-file-sync'`

Reply:
That shouldn't be a breaking error, but you can also probably do `yarn add -D @types/download-file-sync`